src/static/css/cellml.css

```diff
@@ -32,6 +36,10 @@ div[class^="issue"], div[class*=" issue"] > ol {
     margin-bottom: 0;
 }
 
+li {
+    margin-bottom:12px;
```

Comment:
Space after the column (same on line 13 and maybe elsewhere, I haven't checked).

Reply:
0759f91 I've done a quick tidy.  A lot of these classes could be consolidated, but I'm going to leave that for the end once we know about local styles etc.